src/clusterfuzz/_internal/tests/core/bot/tasks/utasks/corpus_pruning_task_test.py

```diff
@@ -439,30 +435,20 @@ def tearDown(self):
   def test_prune(self):
     """Test pruning."""
     self._setup_env(job_type='libfuzzer_asan_job')
-    self.mock._record_cross_pollination_stats.side_effect = (
```

Comment:
I'm not sure if this check ever passed, it's failures are silent because after they happen a `catch Exception` in corpus_pruning_task hides the error.